src/main/java/com/opencart/pages/EditAccountPage.java

```diff
@@ -0,0 +1,14 @@
+package com.opencart.pages;
+
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public class EditAccountPage extends BasePage {
```

Comment:
Could you, please, rename it to LeftSideMenuOptionsPage or something like that? because it is not only related to EditAccount, there are also such options like WishList, Password, etc.

Reply:
Please, make changes according to this comment